src/Model/Site.php

```diff
@@ -231,7 +231,11 @@ public function setMetaKeywords($metaKeywords): void
     public function getUrl()
     {
         if ($this->isLocalhost()) {
-            return $this->getRelativePath();
+            $relativePath = $this->getRelativePath();
```

Comment:
When relativePath is null, it will be used in the sprintf and it wont make sens.

Also, the sprintf shouldnt be used if isLocalhost.

I think we should throw a logicException if the relativePath is null (or used an assert if we're not sure).

Or allowing to return null (or returning '').

Reply:
I went for allowing null here.